src/json/json.go

```diff
@@ -15,7 +15,7 @@ func ReadFile(dir string, relPath string, target interface{}, errPrefix string)
 	content, err := os.ReadFile(path)
 	if err != nil {
 		if os.IsNotExist(err) {
-			return fmt.Errorf("%s file \"%s\" not found", errPrefix, relPath)
+			return fmt.Errorf("missing %s file \"%s\"", errPrefix, relPath)
```

Comment:
fixnuta hlaska, `not found` na konci hlasky sa blbo cita ak je `path` dlha, a nebolo to konzistentne s ostatnými hlaskami.